src/graphics/program-lib/programs/standard.js

```diff
@@ -371,6 +383,14 @@ const standard = {
                 code.append(this._addMap("light", lightmapChunkPropName, options, litShader.chunks, options.lightMapEncoding));
                 func.append("getLightMap();");
             }
+
+            // only add the legacy chunk if it's referenced
+            if (code.code.indexOf('texture2DSRGB') !== -1 ||
```

Comment:
could this be tracked differently than using .indexOf?
Likely not I assume, as any chunk can just call this directly.

Reply:
Yeah this is no nasty :`(

Alternatively since only emissive and lightmap chunks do this, we could somehow pass flags around tracking the use of these functions instead. This code is just so much simpler.